From cdb35ea867fa62f5c65655d5a577e00fbb94f5b3 Mon Sep 17 00:00:00 2001
From: Giuseppe Lippolis <giu.lippolis@gmail.com>
Date: Thu, 16 Mar 2023 09:32:35 +0530
Subject: [PATCH 004/500] OpenWrt: 130-add-linux-spidev-compatible-si3210.patch

Add the linux,spidev compatible in spidev Several device in ramips have this binding in the dts

Signed-off-by: Giuseppe Lippolis <giu.lippolis@gmail.com>

Change-Id: Ic39209763fee59c713911ce7c9a79bfd8b4e353b
Signed-off-by: Ram Chandra Jangir <quic_rjangir@quicinc.com>
(cherry picked from commit efced67dd53073b67d1b7aa1e6aad861ddaf5b5e)
---
 drivers/spi/spidev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/spi/spidev.c b/drivers/spi/spidev.c
index d13dc15cc191..f8efe21e19e9 100644
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -742,6 +742,7 @@ static const struct of_device_id spidev_dt_ids[] = {
 	{ .compatible = "semtech,sx1301", .data = &spidev_of_check },
 	{ .compatible = "silabs,em3581", .data = &spidev_of_check },
 	{ .compatible = "silabs,si3210", .data = &spidev_of_check },
+	{ .compatible = "siliconlabs,si3210" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
-- 
2.34.1

